exp/runtime/hooks/api/v1alpha1/util.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
nit: let's try to rename/split up this file a bit generic util.go is always dangerous because we just drop everything into it over time